packages/web3-core/src/types.ts

```diff
@@ -48,8 +48,11 @@ export interface Web3ConfigOptions {
 	transactionPollingTimeout: number;
 	blockHeaderTimeout: number;
 	maxListenersWarningThreshold: number;
```

Comment:
like 1.x If provider does not support event subscription, we should use `transactionPollingInterval` configurable via config options here?

```suggestion
    maxListenersWarningThreshold: number;
	transactionPollingInterval: number;
```

Reply:
Sure, not relevant to this PR though - created an [issue](https://github.com/ChainSafe/web3.js/issues/4764)